packages/std/src/math/uint128.rs

```diff
@@ -502,6 +508,30 @@ mod tests {
     use super::*;
     use crate::{from_slice, to_vec};
 
+    #[test]
+    fn uint128_one_works() {
+        let one = Uint128::one();
+        assert_eq!(
+            one.to_be_bytes(),
+            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1]
+        );
+    }
+
+    #[test]
+    fn uint128_zero_and_one_works() {
+        let zero = Uint128::zero();
+        assert_eq!(
+            zero.to_be_bytes(),
+            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,]
+        );
+
+        let one = Uint128::one();
+        assert_eq!(
+            one.to_be_bytes(),
+            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1]
+        );
```

Comment:
I had the `::one()` tests in here (`uint128_zero_and_one_works`). But this is probably easy to miss. Will split in two tests.